UnicornRender/video/include/unicorn/video/vulkan/VkMesh.hpp

```diff
@@ -55,47 +57,88 @@ class VkMesh
      * @brief Allocation on GPU
      */
     void AllocateOnGPU();
+
     /**
      * @brief Deallocation on GPU
      */
     void DeallocateOnGPU();
+
     /**
      * @brief Returns vertex buffer
      * @return vulkan buffer
      */
-    vk::Buffer GetVertexBuffer();
+    vk::Buffer GetVertexBuffer() const;
+
     /**
      * @brief Returns index buffer
      * @return vulkan buffer
      */
-    vk::Buffer GetIndexBuffer();
+    vk::Buffer GetIndexBuffer() const;
     /**
      * @brief Matrix of transformations
      * @return model matrix
      */
-    const glm::mat4& GetModel() const;
+    const glm::mat4& GetModelMatrix() const;
+
     /**
      * @brief Returns vertices size
      * @return size of vertices
      */
-    uint32_t VerticesSize();
+    uint32_t VerticesSize() const;
+
     /**
     * @briefReturns indices size
     * @return size of indices
     */
-    uint32_t IndicesSize();
+    uint32_t IndicesSize() const;
+
+    /**
+     * @brief Checks if needed to be rendered only in color
+     * @return true if mesh is colored and false if not
+     */
+    bool IsColored() const;
+
+    /**
+     * @brief Checks if wireframe mode is enabled for this mesh
+     * @return true if mesh needs to be rendered in wireframe mode, false if not
```

Comment:
The whole thing can be shortened to `/** Returns @c true if mesh is in wired mode and @c false otherwise */`